plugin/evm/vm_database.go

```diff
@@ -0,0 +1,82 @@
+// (c) 2019-2021, Ava Labs, Inc. All rights reserved.
```

Comment:
I suppose we want to keep it as a full copy-paste right?
Otherwise we might want to bump the end year:

```suggestion
// (c) 2019-2025, Ava Labs, Inc. All rights reserved.
```

Reply:
Yes let's leave them the same in both codebases